tests/components/drop_connect/common.py

```diff
@@ -216,3 +221,27 @@ def config_entry_ro_filter() -> ConfigEntry:
         },
         version=1,
     )
+
+
+def help_assert_entries(
+    hass: HomeAssistant,
+    entity_registry: er.EntityRegistry,
+    snapshot: SnapshotAssertion,
+    config_entry: ConfigEntry,
+    step: str,
+    assert_unknown: bool = False,
+) -> None:
+    """Assert platform entities and state."""
+    entity_entries = er.async_entries_for_config_entry(
+        entity_registry, config_entry.entry_id
+    )
+    assert entity_entries
+    if assert_unknown:
+        for entity_entry in entity_entries:
+            assert hass.states.get(entity_entry.entity_id).state == STATE_UNKNOWN
+        return
+
+    for entity_entry in entity_entries:
+        assert hass.states.get(entity_entry.entity_id).state == snapshot(
```

Comment:
```suggestion
        assert hass.states.get(entity_entry.entity_id) == snapshot(
```
This would capture the full state including UoM etc